cd/dev.env

```
@@ -0,0 +1,4 @@
+ENP_ALGORITHM=foo
+ENP_ACCESS_TOKEN_EXPIRE_SECONDS=bar
+MAX_RETRIES=foo
```

```suggestion
ENP_ALGORITHM=HS256
ENP_ACCESS_TOKEN_EXPIRE_SECONDS=60
MAX_RETRIES=2886
```

Could you please add the values for these?